eclipse-scout-core/src/layout/graphics.ts

```diff
@@ -208,6 +213,7 @@ export const graphics = {
     }
 
     // modify properties which prevent reading the preferred size
+    $elem.addClass('measure');
```

Comment:
Maybe remove measure class in DesktopNavigationLayout.ts because this makes it obsolete